libtextsecure/outgoing_message.js

```diff
@@ -30,6 +30,28 @@ const getTTLForType = type => {
   }
 };
 
+const DebugMessageType = {
+  AUTO_FR_REQUEST: 'auto-friend-request',
+  AUTO_FR_ACCEPT: 'auto-friend-accept',
+
+  SESSION_REQUEST: 'session-request',
+  SESSION_REQUEST_ACCEPT: 'session-request-accepted',
+
+  SESSION_RESET: 'session-reset',
+  SESSION_RESET_RECV: 'session-reset-received',
+
+  OUTGOING_FR_ACCEPTED: 'outgoing-friend-request-accepted',
+  INCOMING_FR_ACCEPTED: 'incoming-friend-request-accept',
+
+  REQUEST_SYNC_SEND: 'request-sync-send',
+  CONTACT_SYNC_SEND: 'contact-sync-send',
+  CLOSED_GROUP_SYNC_SEND: 'closed-group-sync-send',
+  OPEN_GROUP_SYNC_SEND: 'open-group-sync-send',
+
+  DEVICE_UNPAIRING_SEND: 'device-unpairing-send',
+  PAIRING_REQUEST_SEND: 'pairing-request',
+};
+
```

Comment:
basically a message type used for debugging purposes.
not complete. probably a better way to handle this